samples/matter/template/README.rst

```diff
@@ -135,13 +135,13 @@ Device Firmware Upgrade support
 
 Alternatively, for the nRF54L15 DK, the DFU can be configured to only use the internal MRAM for storage.
 This means that both the currently running firmware and the new firmware to be updated will be stored within the device's internal flash memory.
-This configuration is only available for the :ref:`release configuration <matter_template_custom_configs>`.
+This configuration is by default enabled for the :ref:`debug configuration <matter_template_custom_configs>`.
```

Comment:
```suggestion
This configuration is enabled by default for the :ref:`debug configuration <matter_template_custom_configs>`.
```
Going to nitpick here, just because I can.